var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -158,6 +156,7 @@ class Acts(CMakePackage, CudaPackage):
     depends_on('py-pytest', when='+python +unit_tests')
     depends_on('root @6.10: cxxstd=14', when='+tgeo @:0.8.0')
     depends_on('root @6.20: cxxstd=17', when='+tgeo @0.8.1:')
+    depends_on('vecmem@0.4: +sycl', when='+sycl')
```

Comment:
Since ACTS [also has a `find_package(SYCL)`](https://github.com/acts-project/acts/blob/8ce4d313847e825b1e67f16bd8a5eba2ab5ad422/CMakeLists.txt#L216), you will also want a `depends_on('sycl', when='+sycl')`.